gettsim/parameters/eink_st.yaml

```diff
@@ -228,12 +228,12 @@ ertragsanteil:
     1:
       lower_threshold: 2005
       upper_threshold: 2021
-      rate_linear: 0.02
+      rate_linear: -0.02
```

Comment:
Could you explain shortly the rational behind the minus?

Reply:
because it yields the correct rates :man_shrugging: 

```python
from gettsim.policy_environment import set_up_policy_environment
from gettsim.taxes.lohn_st import vorsorg_rv_anteil
policy_params, policy_functions = set_up_policy_environment("2021")
print(vorsorg_rv_anteil(policy_params['eink_st_abzuege']))
policy_params, policy_functions = set_up_policy_environment("2010")
print(vorsorg_rv_anteil(policy_params['eink_st_abzuege']))
```
yields `0.84` and `0.40`, respectively, which is correct